src/crypto-providers/util.ts

```diff
@@ -424,21 +424,26 @@ const getAddressParameters = (
   network: Network,
 ): _AddressParameters | null => {
   const addressType = getAddressType(address)
-  try {
-    switch (addressType) {
-      case AddressTypes.BOOTSTRAP:
-        return _packBootStrapAddress(hwSigningData[0], network)
-      case AddressTypes.BASE:
-        return _packBaseAddress(hwSigningData, network)
-      case AddressTypes.ENTERPRISE:
-        return _packEnterpriseAddress(hwSigningData[0], network)
-      case AddressTypes.REWARD:
-        return _packRewardAddress(hwSigningData, network)
-      default: return null
+  const packAddress = () => {
```

Comment:
It's odd to me to have a function without arguments here, what about extracting it into a "global" `_packAddress`?

Reply:
See comment above